### DISCUSSION TITLE
Dynamically modify country select form field

### DISCUSSION THREAD

**Discussion by Slawikus** [SmileyChris/django-countries#374] [Q&A]
Hi there!

Is there any way to dynamically modify country select choices. My use case - if user have already selected a country I don't want to show this country anymore in the select options. The only way to customise country list I see in the documentation is:
```
class G8Countries(Countries):
    only = ['CA', 'FR', 'DE', 'IT', 'JP', 'RU', 'GB']

class VisitedCountryForm(ModelForm):
    country = CountryField(countries=G8Countries).formfield()
```
but there is no way to make it dynamic.

Any suggestion on how to proceed with this?

**Comment by SmileyChris** [comment]
At the end of the day, you've got a subclass of a ChoiceField form field. You should be able to mess with the choices it generates in any dynamic way you want.
